src/components/icon/list.tsx

```diff
@@ -611,21 +612,6 @@ export function getIcon(
           />
         </svg>
       );
-    case IconName.arrow_down_filled:
```

Comment:
this one is used in `src/pages/project/file-browser/BreadCrumbsBox/FileActionsButton.tsx`
you've changed it in PR, right?

Reply:
no this was a duplicate, basically never reachable redundant code